xarray/core/formatting_html.py

```diff
@@ -184,7 +185,7 @@ def dim_section(obj):
 def array_section(obj):
     # "unique" id to expand/collapse the section
     data_id = "section-" + str(uuid.uuid4())
-    collapsed = "checked"
+    collapsed = "" if OPTIONS["collapse_html"] else "checked"
```

Comment:
I think you are right that this collapsed var is misnamed. It would be clearer to rename it to "expanded". The `collapsible_section` also has this inverted name.

Reply:
i can rename it in the array section but I guess renaming it in the `collapsible_section` leaves the scope of this issue?

Reply:
problem is that for `collapsible_section` it's a function parameter, so it would need to be changed in all calling cases as well for clarity?